keyword_clustering.py

```diff
@@ -27,25 +27,23 @@ def main(args):
     shortened_keywords_list = []
     total_keyword_counts = 0
     
-    with open(input_file_path) as f:
-        read_tsv = csv.reader(f, delimiter="\t") 
-        model_name_list = next(read_tsv)[1:]
-        for i in range(len(model_name_list)):
-            shortened_keywords_list.append({})
-            
-        for line in read_tsv:
-            total_keyword_counts += 1
-            for index, shortened_keyword in enumerate(line[1:]):
-                shortened_keyword = shortened_keyword.lower()
-                if shortened_keyword == "":
-                    continue
-                if shortened_keyword not in shortened_keywords_list[index]:
-                    shortened_keywords_list[index][shortened_keyword] = [line[0]]
-                else:
-                    shortened_keywords_list[index][shortened_keyword].append(line[0])
-    
-    if len(model_name_list) == 1:
-        model_name_list = [""]
+    f = open(input_file_path)
+    read_tsv = csv.reader(f, delimiter="\t") 
+    model_name_list = next(read_tsv)[1:]
+    for i in range(len(model_name_list)):
+        shortened_keywords_list.append({})
+
+    for line in read_tsv:
+        total_keyword_counts += 1
+        for index, shortened_keyword in enumerate(line[1:]):
+            shortened_keyword = shortened_keyword.lower()
+            if shortened_keyword == "":
+                continue
+            if shortened_keyword not in shortened_keywords_list[index]:
+                shortened_keywords_list[index][shortened_keyword] = [line[0]]
+            else:
+                shortened_keywords_list[index][shortened_keyword].append(line[0])
+    f.close()
```

Comment:
same here, the with statement is good, you can just keep it.

Reply:
Done